test/mocks/v1/testing_snapshot.sk.go

```diff
@@ -257,6 +257,65 @@ func (s *TestingSnapshot) RemoveFromResourceList(resource resources.Resource) er
 	}
 }
 
+func (s *TestingSnapshot) RemoveAllResourcesInNamespace(namespace string) {
+	var Simplemocks SimpleMockResourceList
```

Comment:
It feels like this is a specific implementation of a Map/Reduce type function. What if instead of just accepting a namespace, we have a function
```suggestion
	func (s *TestingSnapshot) RemoveMatches(predicate func(core.Metadata) {
```

And then the predicate we pass in is a function that inspects the metadata.namespace?

Reply:
Yeah this is a better idea. Let me update this